Sounds/wwise/Init.txt

```diff
@@ -0,0 +1,7 @@
+Audio Bus	ID	Name			Wwise Object Path	Notes
```

Comment:
These *.txt files in the banks folder technically aren't necessary to add either because they can be generated by Wwise, but I'm less concerned about them because they don't point at anything like hard-coded user paths.

Reply:
Done, thanks!